library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -152,10 +153,14 @@ class Client() {
 
         fun canMessage(peerAddress: String, options: ClientOptions? = null): Boolean {
             val clientOptions = options ?: ClientOptions()
-            val api = GRPCApiClient(
-                environment = clientOptions.api.env,
-                secure = clientOptions.api.isSecure,
-            )
+            val v2Client = runBlocking {
```

Comment:
question - is there a reason we use `runBlocking` instead of making canMessage a `suspend` function? I know we previously had more runBlocking, but we I think we pulled them out because we thought we had better performance with using suspend functions in some cases. same would apply to the other runBlocking in this class.